test/e2e/flowaggregator_test.go

```diff
@@ -300,14 +301,18 @@ func testHelper(t *testing.T, data *TestData, isIPv6 bool) {
 	// and their flow information is exported as IPFIX flow records.
 	// K8s network policies are being tested here.
 	t.Run("IntraNodeFlows", func(t *testing.T) {
-		checkIntraNodeFlows(t, data, podAIPs, podBIPs, isIPv6)
+		label := "IntraNodeFlows"
+		addLabelToPerftestPods(t, data, label)
```

Comment:
I think it may be a good idea to add a comment explaining why we label the Pods here, but we don't need to repeat the comment for every test case

Reply:
Added. Thanks.